### ISSUE TITLE
Getting started with a new project

### ISSUE THREAD

**Issue by benhalverson** [fastify/fastify-cli#89]
Expected 
`fastify generate` creates a new project as the docs state.

Actual result
```
--> reading package.json in /Users/benhalverson/projects/fastify-cli-test
ENOENT: no such file or directory, open '/Users/benhalverson/projects/fastify-cli-test/package.json'
```

The workaround is to create a package.json file first in directory your project will be in
`npm init -y`

**Comment by delvedor** [comment]
As [documented](https://github.com/fastify/fastify-cli#generate) `fastify generate` expects that the `package.json` is already created, so this is not an issue.

@mcollina should we think about create the `package.json` is is not present?
In which case probably we should also create the directory.

**Comment by cemremengu** [comment]
Landed in #114 (v0.25.0)
